ncm-symlink/src/main/perl/symlink.pod

```diff
@@ -28,13 +28,62 @@ Target definition and contextual variable value may also contain expression inte
 
 A list of symbolic links to create or delete.  Each entry
 must be of the structure_symlink_entry type which has the following
-fields: name, target, delete, exists, replace.
+fields: 
 
-'name' is the symbolic link name, 'target' is the link target, other fields are boolean. 'delete' means delete the link rather than create. 'exists' means check the target exists when creating or check the link exists when deleting. 
+=over
 
-'replace' is an option taking one or several of the several following suboptions : all, dir, dirempty, file, link, none. Each suboption can have the following values : 'yes', 'no', a string interpreted as a file extension used to rename the existing object with the link name. If a suboption value is empty, it is interpreted as 'yes'. If value is 'yes', replacement is enabled but existing object is not renamed, except if a default extension has been defined with 'none' or 'all' or if it is a non empty directory (renamed with a default extension).
+=item *
 
-When 'all' or 'none' suboption is specified, other suboptions can be specified too to redefine extension or to prevent replacement of a specific object type. 'dirempty' means directory is replaced only if empty. 'none=extension' can be used to establish a default backup extension without enabling replacement (as it is with 'all').
+C<name>: symbolic link name
+
+=item *
+
+C<target>: link target path
+
+=item *
+
+C<delete>: (boolean) delete the symlink rather than creating it 
+
+=item *
+
+C<exists>: (boolean) check that the target exists when creating it or check that the link exists when deleting it
+
+=item *
+
+C<replace>: (nlist) option used to specify the action to do when an object with the same name as the symlink already exists, depending on the object type. 
```

Comment:
`action to do` -> `action to take`? (what happens if not set and name exists)
`an object with the same name as the symlink` you mean if the name already exists, and what sort of objects are defined: existing links, or files and/or directories? and is there a backup option, esp if you support replacing files/directories?